lldb/include/lldb/Core/Progress.h

```diff
@@ -97,12 +98,32 @@ class Progress {
   /// Used to indicate a non-deterministic progress report
   static constexpr uint64_t kNonDeterministicTotal = UINT64_MAX;
 
+  /// Use a struct to send data from a Progress object to
+  /// ProgressManager::ReportProgress. In addition to the Progress member fields
```

Comment:
I think the first sentence isn't entirely accurate. The struct is used to store information about the progress event in the progress manager?

Reply:
Yes, this can be generalized more since we use the struct for more than just the call to `ReportProgress`.